python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -206,6 +206,53 @@ def my_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource, config: MyDb
                     dbt_build_args += ["--full-refresh"]
 
                 yield from dbt.cli(dbt_build_args, context=context).stream()
+
+        Creating partitions for dbt assets:
+
+        :any:`dbt_assets` produces multiple assets, but the :any:`asset_partitions_time_window_for_output`
+        method expects a single output. Since all dbt assets have the same
+        partition definition, we can provide the first dbt asset to the method
+        in order to retrieve the first and last partition window.
+
+        We pass these as variables to the dbt CLI which can be read as jinja
+        macro variables.
+
+        .. code-block:: python
+
+            import json
+
+            from dagster import OpExecutionContext, DailyPartitionsDefinition
+            from dagster_dbt import DbtCliResource, dbt_assets
+
+            daily_partitions = DailyPartitionsDefinition(start_date="2023-01-01")
+
+
+            @dbt_assets(manifest=Path("target", "manifest.json"), partitions_def=daily_partitions)
+            def partitionshop_dbt_assets(context: OpExecutionContext, dbt: DbtCliResource):
+                first, last = context.asset_partitions_time_window_for_output(
```

Comment:
From the method name, it's not immediately obvious that this can be deconstructed.
```suggestion
                time_window = context.asset_partitions_time_window_for_output(
```